tasks/in-house-ctf/template.py

```diff
@@ -0,0 +1,12 @@
+from string import Template
```

Comment:
I think we should roll `template.py` into `task.py` (as its so short and its in general helpful to see the system prompt when looking at the task source code)